src/librawspeed/decompressors/VC5Decompressor.h

```diff
@@ -0,0 +1,125 @@
+/*
+    RawSpeed - RAW file decoder.
+
+    Copyright (C) 2018 Stefan Löffler
+
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2 of the License, or (at your option) any later version.
+
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this library; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+*/
+
+#pragma once
+
+#include "common/Common.h"                      // for uint32
+#include "common/RawImage.h"                    // for RawImageData
+#include "io/ByteStream.h"                      // for ByteStream
+#include "io/BitPumpMSB.h"                      // for BitPumpMSB
+#include "decompressors/AbstractDecompressor.h" // for AbstractDecompressor
+
+namespace rawspeed {
+
+const int MAX_NUM_CHANNELS = 4;
+const int MAX_NUM_WAVELETS = 3;
+const int MAX_NUM_BANDS = 4;
+const int MAX_NUM_SUBBANDS = 10;
+const int MAX_NUM_PRESCALE = 8;
+
+class ByteStream;
+class RawImage;
+
+// Decompresses VC-5 as used by GoPro
+
+#define VC5_LOG_TABLE_SIZE 4096
+
+class VC5Decompressor final : public AbstractDecompressor
+{
```

Comment:
Sorry, I don't normally use clang, and don't have that tool installed by default (and did not know I should have, either). I will look into it if required.

Reply:
`clang-format` is standalone, so you wouldn't even need to use clang.
Really, everyone should be using it this late in this decade.

It may be best for you to format the new files (git blame reasons :))